docs/alpha-features.md

```diff
@@ -0,0 +1,29 @@
+- [Alpha Table](#alpha-features-table)
+
+## Introduction
+
+This doc provides a list of features in Tekton Pipelines that are considered [Alpha](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#alpha-beta-and-ga).
+
+Set `enable-api-fields` to `"alpha"` to allow alpha features to be used.
+
+## Alpha Features Table
+
+| Feature                                                                                                                         | TEP                                                                                         | Release                                                              | Individual Flag             |
+|:------------------------------------------------------------------------------------------------------------------------------- |:------------------------------------------------------------------------------------------- |:-------------------------------------------------------------------- |:--------------------------- |
+| [Bundles ](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#tekton-bundles)                                  | [TEP-0005](https://github.com/tektoncd/community/blob/main/teps/0005-tekton-oci-bundles.md) | [v0.18.0](https://github.com/tektoncd/pipeline/releases/tag/v0.18.0) | `enable-tekton-oci-bundles` |
+| [`Runs` and `Custom Tasks`](https://github.com/tektoncd/pipeline/blob/main/docs/runs.md)                                        | [TEP-0002](https://github.com/tektoncd/community/blob/main/teps/0002-custom-tasks.md)       | [v0.19.0](https://github.com/tektoncd/pipeline/releases/tag/v0.19.0) | `enable-custom-tasks`       |
+| [Isolated `Workspaces`](https://github.com/tektoncd/pipeline/blob/main/docs/workspaces.md#isolated-workspaces)                  | [TEP-0029](https://github.com/tektoncd/community/blob/main/teps/0029-step-workspaces.md)    | [v0.24.0](https://github.com/tektoncd/pipeline/releases/tag/v0.24.0) |                             |
+| [Hermekton](https://github.com/tektoncd/pipeline/blob/main/docs/hermetic.md)                                                    | [TEP-0025](https://github.com/tektoncd/community/blob/main/teps/0025-hermekton.md)          | [v0.25.0](https://github.com/tektoncd/pipeline/releases/tag/v0.25.0) |                             |
+| [Specifying `onError` for a `Step`](https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#specifying-onerror-for-a-step) | [TEP-0040](https://github.com/tektoncd/community/blob/main/teps/0040-ignore-step-errors.md) | [v0.27.0](https://github.com/tektoncd/pipeline/releases/tag/v0.27.0) |                             |
+| [Implicit `Parameters`](https://github.com/tektoncd/pipeline/blob/main/docs/taskruns.md#implicit-parameters)                    | [TEP-0023](https://github.com/tektoncd/community/blob/main/teps/0023-implicit-mapping.md)   | [v0.28.0](https://github.com/tektoncd/pipeline/releases/tag/v0.28.0) |                             |
+
```

Comment:
PipelineRun Timeouts, Graceful Termination, and Windows Script support are all covered by alpha as well I think.

Relevant doc links:

- [PipelineRun Timeouts](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#configuring-a-failure-timeout)
- [Graceful Termination](https://github.com/tektoncd/pipeline/blob/main/docs/pipelineruns.md#gracefully-cancelling-a-pipelinerun)
- [Windows Scripts](https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#windows-scripts)

🤞 hopefully didn't miss any others - I caught these by searching for "ValidateEnabledAPIFields" in our validation code.